backend/eatandmeet/api/models.py

```diff
@@ -25,11 +26,20 @@ class Event(models.Model):
     event_members = models.ManyToManyField(User, related_name='event_members', blank=True)
     topics = MultiSelectField(choices=TOPIC_CHOICES)
     location = models.CharField(max_length=11, choices =PLACE_CHOICES, default ='Hauptmensa' )
```

Comment:
Also remove the voices here. Let it be just a string.

```suggestion
    location = models.CharField(max_length=30, default ='Hauptmensa' )
```